x11/pango/Portfile

```diff
@@ -91,10 +90,14 @@ if {[variant_isset x11]} {
     require_active_variants path:lib/pkgconfig/cairo.pc:cairo x11
 }
 
+pre-configure {
```

Comment:
Maybe add a note about why this is required, since this isn’t needed for newer versions of glib

Reply:
@i0ntempest Zhenfu, any thoughts?

Reply:
This might be a mistake that I forgot to remove, it was used to force meson to not build glib as a subproject. Since we now have an updated glib this should be able to be removed without problems.

Reply:
@i0ntempest Isn't that also the case for `-Wno-error,-Wimplicit-fallthrough` as I’ve not needed that for my updated version of pango.

Reply:
That cflag was indeed needed to build on my system when I tried, did another version came out? If yes it might not be needed anymore.

Reply:
I have glib-2.68.4 installed on my end and haven't needed this cflag on Mojave